pkg/ebpf/ebpftest/buildmode_linux.go

```diff
@@ -24,7 +25,7 @@ func init() {
 
 func SupportedBuildModes() []BuildMode {
 	modes := []BuildMode{Prebuilt, RuntimeCompiled, CORE}
-	if runtime.GOARCH == "amd64" && (hostinfo.Platform == "amazon" || hostinfo.Platform == "amzn") && kv.Major() == 5 && kv.Minor() == 10 {
+	if os.Getenv("TEST_FENTRY_OVERRIDE") == "true" || (runtime.GOARCH == "amd64" && (hostinfo.Platform == "amazon" || hostinfo.Platform == "amzn") && kv.Major() == 5 && kv.Minor() == 10) {
```

Comment:
This only used to determine which modes to run the tests in right ?

It feels like poor devx that in some environments, the fentry tests run if available, and on others you have to force it. 

Can we update this method to intelligently enable fentry in our development environments?

Reply:
This is how it is right now with the CI ... since this is an Amazon only thing. Since we don't have dev vms with Amazon Linux, this is the only way to enable these tests in dev environments.